event-schema/event_schema.go

```diff
@@ -1083,16 +1083,16 @@ func (manager *EventSchemaManagerT) Setup() {
 	eventSchemaChannel = make(chan *GatewayEventBatchT, 10000)
 
 	for i := 0; i < noOfWorkers; i++ {
-		rruntime.Go(func() {
+		rruntime.GoForWarehouse(func() {
 			manager.recordEvents()
 		})
 	}
 
-	rruntime.Go(func() {
+	rruntime.GoForWarehouse(func() {
```

Comment:
Instead of specific function for Warehouse, can we do one of the following two:
1. Add an argument for module
2. GoForModule(func, module)

Reply:
Thanks for the feedback. @snarkychef Will have this change made in another pr.